thumbor/optimizers/gifv.py

```diff
@@ -8,114 +8,15 @@
-                )
-                return False
-            return True
-        return False
-
-    def optimize(self, image_buffer, input_file, output_file):
-        file_format, command_params = self.set_format()
-
-        bg_color_hex = None
-        if "background_color" in self.context.request.filters:
-            filters = self.context.request.filters.split(":")
-            bg_filter = [
-                filter
-                for filter in filters
-                if filter.startswith("background_color")
-            ][0]
-            bg_color = re.search(r"\((.*?)\)", bg_filter).group(1)
-            bg_color_hex = self.normalize_color_to_hex(bg_color)
-
-        command = [
-            self.context.config.FFMPEG_PATH,
-            "-y",
-            "-f",
-            "gif",
-            "-trans_color",
-            f"0xff{(bg_color_hex[1:] if bg_color_hex else 'ffffff')}",
-            "-i",
-            input_file,
-            "-filter_complex",
-            "scale=trunc(iw/2)*2:trunc(ih/2)*2",
-            "-an",
-            "-movflags",
-            "faststart",
-            "-f",
-            file_format,
-            "-pix_fmt",
-            "yuv420p",
-        ]
-
-        command += command_params
-
-        command += [
-            "-qmin",
-            "10",
-            "-qmax",
-            "42",
-            "-crf",
-            "23",
-            "-maxrate",
-            "500k",
-            output_file,
-            "-loglevel",
-            "error",
-        ]
-
-        subprocess.call(command, stdin=subprocess.DEVNULL)
-
-        self.context.request.format = file_format
-
-    def set_format(self):
-        if "webm" in self.context.request.filters:
-            file_format = "webm"
-            command_params = [
-                "-quality",
-                "good",
-                "-cpu-used",
-                "4",
-                "-b:v",
-                "500k",
-            ]
-        else:
-            file_format = "mp4"
-            command_params = ["-profile:v", "baseline", "-level", "4.0"]
-        return file_format, command_params
-
-    def normalize_color_to_hex(self, color_string):
-        try:
-            return webcolors.normalize_hex("#" + color_string)
-        except ValueError:
-            pass
-
-        try:
-            return webcolors.name_to_hex(color_string)
-        except ValueError:
-            pass
-
-        try:
-            return webcolors.normalize_hex(color_string)
-        except ValueError:
-            pass
-
-        if color_string:
-            logger.error("background_color value could not be parsed")
-
-        return None
+class Optimizer(gifv.Optimizer):
+    logger.warning("""
+    This optimizer will be deprecated on the next major version.
+    Add thumbor-plugins-gifv as a dependency, and
+    change thumbor.optimizers.gifv to thumbor_plugins.optimizers.gifv
+    on the OPTIMIZERS list on you thumbor.conf.
+    """)
```

Comment:
```suggestion
    """
    )
```

Reply:
This code doesn't look to me like it was *blackified*. We were *blackifying* all code, IIRC. I'd say we just accidentaly stopped checking it as part of the workflow. If that's really the case, I will happily fix it in a PR.